src/main/java/practicalities/items/ItemMagnet.java

```diff
@@ -64,6 +67,8 @@ public void onUpdate(ItemStack item, World world, Entity entity, int i, boolean
 				this.distanceFromPlayer).iterator();
 		while (iterator.hasNext()) {
 			EntityItem itemToGet = (EntityItem) iterator.next();
+			if( PolarityNegatorManager.isEntityCloseToNegator(itemToGet) )
```

Comment:
Oh, yeah. Never mind

Sent from Nine

From: thecodewarrior <notifications@github.com>
Sent: Nov 26, 2015 8:27 PM
To: joekaiser/practicalities
Cc: Joe
Subject: Re: [practicalities] Add polarity negator (#16)

> @@ -64,6 +67,8 @@ public void onUpdate(ItemStack item, World world, Entity entity, int i, boolean
>                               this.distanceFromPlayer).iterator();
>               while (iterator.hasNext()) {
>                       EntityItem itemToGet = (EntityItem) iterator.next();
> +                     if( PolarityNegatorManager.isEntityCloseToNegator(itemToGet) )

That might work, though most magnets pull items, so they would be pulled out of the range.

---
Reply to this email directly or view it on GitHub:
https://github.com/joekaiser/practicalities/pull/16/files#r46016722